Packs/Okta/Integrations/OktaLog/OktaLog.py

```diff
@@ -0,0 +1,183 @@
+            if int(self.client.request.params.limit) == 0 or len(events) == 0:  # type: ignore
+                break
+        return stored_events
+
+    @staticmethod
+    def get_last_run(events: List[dict]) -> dict:
+        """
+        Get the info from the last run, it returns the time to query from and a list of ids to prevent duplications
+        """
+
+        ids = []
+        # gets the last event time
+        last_time = events[-1].get('published')
+        for event in reversed(events):
+            if event.get('published') != last_time:
+                break
+            ids.append(event.get('uuid'))
+        last_time = datetime.strptime(str(last_time).lower().replace('z', ''), '%Y-%m-%dt%H:%M:%S.%f')
+        return {'after': last_time.isoformat(), 'ids': ids}
+
+    @staticmethod
+    def remove_duplicates(events: list, ids: list) -> list:
+        """
+        Remove object duplicates by the uuid of the object
+        """
+        return [event for event in events if event['uuid'] not in ids]
+
+
+def main():
+    try:
+        demisto_params = demisto.params() | demisto.args()
+        events_limit = int(demisto_params.get('limit', 2000))
+        after = dateparser.parse(demisto_params['after'].strip())
+        api_key = demisto_params['api_key']['credentials']['password']
+        demisto_params['headers'] = {"Accept": "application/json", "Content-Type": "application/json",
+                                     "Authorization": f"SSWS {api_key}"}
+        last_run = demisto.getLastRun()
+        last_object_ids = last_run.get('ids')
+        if 'after' not in last_run:
+            delta = (datetime.today() - after)
+            delta = delta.total_seconds()
+            delta = datetime.today() - timedelta(seconds=delta)
+            last_run = delta.isoformat()
+        else:
+            last_run = last_run['after']
+        demisto_params['params'] = ReqParams(**demisto_params, since=last_run)
+
+        request = Request(**demisto_params)
+
+        client = Client(request)
+
+        get_events = GetEvents(client)
+
+        command = demisto.command()
+        if command == 'test-module':
+            get_events.aggregated_results()
+            demisto.results('ok')
+        elif command == 'okta-get-events' or command == 'fetch-events':
+            events = get_events.aggregated_results(last_object_ids=last_object_ids)
+            if events:
+                demisto.setLastRun(GetEvents.get_last_run(events))
+                if command == 'okta-get-events':
+                    command_results = CommandResults(
+                        readable_output=tableToMarkdown('Okta Logs', events, headerTransform=pascalToSpace),
+                        outputs_prefix='Okta.Logs',
+                        outputs_key_field='published',
+                        outputs=events,
+                        raw_response=events,
+                    )
+                    return_results(command_results)
```

Comment:
We should still return results in case of "no-events".
```suggestion
            if command == 'okta-get-events':
                command_results = CommandResults(
                    readable_output=tableToMarkdown('Okta Logs', events, headerTransform=pascalToSpace),
                    raw_response=events,
                )
                return_results(command_results)
```